.github/workflows/macos-amd64-build.yml

```diff
@@ -36,11 +36,15 @@ jobs:
         cd ~/work/onnx-mlir
         sh ~/work/onnx-mlir/onnx-mlir/utils/clone-mlir.sh
         sh ~/work/onnx-mlir/onnx-mlir/utils/build-mlir.sh
+    - name: install python requirements
+      run: |
+        cd ~/work/onnx-mlir/onnx-mlir
+        pip3 install --upgrade wheel
```

Comment:
I don't know the details of macOS pip, but it seems odd to use `pip3` here and `python3 -m pip` below.

Reply:
I agree, we should be consistent....maybe we should lean towards using `python3 -m pip`. Seems like python3 -m pip ensures that we are using the python interpreter specified. I think the reason why we are using `python3 -m pip install -v .` on line 48 is to install pip for the `third_party/onnx` directory but if line 43 worked previously....I do not see the need for this.